### ISSUE TITLE
Ability to add apps for users on their behalf

### ISSUE THREAD

**Issue by wsw70** [gotify/server#337]
**Is your feature request related to a problem? Please describe.**

I have users of gotify who are purely on the "receiving side", in the sense that they receive notification and act upon them, but are not technically apt to create their own applications.

On the other hand (and thankfully), whatever application is added to their account (see below) immediately appears on their phone. This is great.

This means that I have to keep the passwords for their users handy, then impersonate them to create an app (and copy the API key)

**Describe the solution you'd like**

An "users admin dashboard" where  I could, from my admin account, create the applications for them (and have access to the API keys which then are used in my applications)

**Describe alternatives you've considered**

Keeping their passwords handy and using incognito mode to connect on their behalf.

**Comment by jmattheis** [comment]
Yeah, this seems like a valid use case. I currently think about removing applications altogether, because I think most of the users don't like the idea and rather have an API like "send message to user with topic xyz". This would need some better permission management but I think this would be the overall better solution.

**Comment by wsw70** [comment]
Yes, I think this would be a great idea, thanks!

**Comment by kaysond** [comment]
I would love to see this feature too!

**Comment by kinderoth** [comment]
I was wondering if there had been any development on this?

**Comment by lxmicode** [comment]
I was wondering if there had been any development on this?

**Comment by jmattheis** [comment]
Nope, no development was done on this ticket.
